css/main.css

```diff
@@ -0,0 +1,498 @@
+/* general */
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  min-height: 100vh;
+  background-color: rgba(236, 236, 236, 0.952);
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+}
+
+main {
+  background-color: white;
+}
+
+/* nav */
+
+.nav {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+  background-color: white;
+  justify-content: space-between;
+  padding-top: 5px;
+  border-bottom: 1px solid rgb(219, 206, 206);
+}
+
+.icons {
+  display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_